common/package.json

```diff
@@ -40,6 +40,7 @@
     "timezone-mock": "^1.3.6",
     "url-template": "^2.0.8",
     "uuid": "^11.1.0",
+    "typed.js": "^2.1.0",
```

Comment:
This was the smallest of the off-the-shelf options I could find ([3kB minified and gziped](https://bundlephobia.com/package/typed.js@2.1.0))